appeals/e2e/cypress/fixtures/appealsApiRequests.js

```diff
@@ -18,7 +18,7 @@ const appealsApiRequests = {
 			knowsAllOwners: 'Some',
 			knowsOtherOwners: 'Some',
 			lpaCode: 'Q9999',
-			nearbyCaseReferences: ['1000000'],
```

Comment:
What is this doing?

Reply:
This will remove a related appeal from the case, at the moment all test cases have this as a related appeal but we need to test adding a related appeal to a case that has no related cases.